backend/tests/Feature/CreateUserEndpointTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature;
+
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class CreateUserEndpointTest extends TestCase
+{
+    use MakesGraphQLRequests;
+    use RefreshDatabase;
```

Comment:
I like having a little separation between functions/classes and use statements. This convention seems to be demonstrated everywhere else in the code too.
```suggestion
    use RefreshDatabase;

```

Reply:
Added in a later commit.

Reply:
Resolved